banners/iota-streams-deprecated.mdx

```diff
@@ -0,0 +1 @@
+This is documentation for Streams, which is no longer actively maintained.
```

Comment:
```suggestion
The for Streams is no longer actively maintained.
```

Reply:
Something is missing here 😆 Do you mean
```suggestion
The documentation for Streams is no longer actively maintained.
```
?
Does that explain well enough that the library will not be maintained? Or only the docs? 😅

Reply:
It's pretty ambiguous in both cases. How about:

```suggestion
Streams and this documentation are no longer actively maintained.
```